index.html

```diff
@@ -115,17 +115,29 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+<<<<<<< Updated upstream
```

Comment:
Perhaps this was injected by your Git interface. Can you remove these lines?

Reply:
Okay.